client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -69,7 +70,7 @@ function ReaderSubscriptionListItem( {
 			</div>
 			<div className="reader-subscription-list-item__options">
 				<FollowButton siteUrl={ siteUrl } followSource={ followSource } />
-				{ isFollowing && <EmailSettings /> }
+				{ isFollowing && isEmailBlocked !== true && <EmailSettings siteId={ siteId } /> }
```

Comment:
User experience wise, it'd be better to not show the cogs and then have them appear when loaded, than to show them briefly and then have them disappear. (The disappearing could be confusing: "Why are my settings disappearing?")